langchain4j-azure-ai-search/src/main/java/dev/langchain4j/store/embedding/azure/search/AzureAiSearchQueryType.java

```diff
@@ -0,0 +1,6 @@
+package dev.langchain4j.store.embedding.azure.search;
+
+public enum AzureAiSearchQueryType {
+
+    VECTOR, FULL_TEXT, HYBRID, HYBRID_WITH_RRF;
```

Comment:
```suggestion
    VECTOR, FULL_TEXT, HYBRID, HYBRID_WITH_RERANKING;
```
My bad.
`HYBRID` also uses RRF to merge the results from full-text and vector searches, so it makes no sense to have `HYBRID` and `HYBRID_WITH_RRF`.
What actually distinguish 2 hybrid types is that the latter [applies re-ranking after RRF](https://learn.microsoft.com/en-us/azure/search/hybrid-search-ranking#diagram-of-a-search-scoring-workflow).